csharp/src/Drivers/Apache/Hive2/HiveServer2Connection.cs

```diff
@@ -90,6 +91,8 @@ internal async Task OpenAsync()
 
         protected internal int HttpRequestTimeout { get; set; } = 30000;
 
+        protected internal int ConnectTimeout { get; set; } = 30000;
```

Comment:
There seem to be a mix of values here. QueryTimeoutDefault looks like seconds, while HttpRequestTimeout, ConnectTimeout appear to be milliseconds. Can we either add a comment or update the name (like PollTimeMillisecondsDefault) to indicate the correct unit?